wallet_presentation/src/commonMain/kotlin/net/thechance/mena/wallet/presentation/screen/transaction_history/component/TransactionFilterBottomSheet.kt

```diff
@@ -80,6 +81,10 @@ fun ScaffoldScope.TransactionFilterBottomSheet(
     )
 }
 
+private fun LocalDate.toUIFormat(): String {
+    return "${this.year}-${this.monthNumber.toString().padStart(2, '0')}-${this.dayOfMonth.toString().padStart(2, '0')}"
+}
```

Comment:
I think the date has to be separated by `\` not `-`, and also this function can be moved to utils package
```suggestion
private fun LocalDate.toUIFormat(): String {
    return "${this.year}\${this.monthNumber.toString().padStart(2, '0')}\${this.dayOfMonth.toString().padStart(2, '0')}"
}
```

Reply:
https://thechance-workspace.slack.com/archives/C09AX4J62LR/p1761744772192779?thread_ts=1761416552.540669&cid=C09AX4J62LR